.bemlintrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Don't add irrelevant changes to the pr, this file has nothing to do with the request star implementation.